crate/operator/backup.py

```diff
@@ -314,7 +319,7 @@ async def create_backups(
 CRONJOB_NAME = "cronjob_name"
 
 
-class EnsureNoBackupsSubHandler(StateBasedSubHandler):
+class BeforeClusterUpdateSubHandler(StateBasedSubHandler):
```

Comment:
I think this no longer fits very well into `backup.py` - maybe `operations.py`? Or we invent a new place...